docs/deployment/README.md

```diff
@@ -217,6 +218,39 @@ $ aws dynamodb create-table \
 
 ---
 
+## Configure the Postgres database
+
+Cumulus requires a Postgres 10.2 compatible database cluster deployed to AWS.    We suggest utilizing [RDS](https://docs.aws.amazon.com/rds/index.html), and have provided a default [template and RDS cluster module](postgres_database_deployment) utilizing Aurora Serverless. However, Core intentionally provides a "bring your own" approach, and any well-planned cluster setup should work, given the following:
+
+- Appropriate testing/evaluation is given to ensure the database capacity will scale and the database deployment will allow access to Cumulus's internal components.   Core provides for security-group oriented permissions management via the `rds_security_group` configuration parameter.
+- The database is configured such that its endpoint is accessible from the VPC and subnets configured for the Core deployment.
+- An AWS Secrets Manager secret exists that has the following format:
+
+```json
+{
+  "database": "databaseName",
+  "host": "xxx",
+  "password": "defaultPassword",
+  "port": 5432,
+  "username": "xxx"
+}
+```
+
+- `database` -- the postgres database used by the configured user
+- `host` -- the RDS service host for the database in the form (dbClusterIdentifier)-(AWS ID string).(region).rds.amazonaws.com
+- `password` -- the database password
+- `port` -- The database connection port, should always be 5432
+- `username` -- the account username
```

Comment:
```suggestion
- `username` -- the database username
```

It should be the username for the database, correct?

Reply:
👍🏻